React/my-app/src/service/books.js

```diff
@@ -1,19 +1,20 @@
 import { api } from "../config/api";
 
-const failure = () => {
-  console.log("Request Failed");
+export const getBooks = url => {
+  return api.get(url);
```

Comment:
Same here, use the url or else the function is the same as api.get (but with less options)